src/server/service/kv.rs

```diff
@@ -160,6 +160,12 @@ impl<T: RaftStoreRouter<E::Local> + 'static, E: Engine, L: LockManager> Service<
         match ch.send_raft_msg(msg) {
             Ok(()) => Ok(()),
             Err(RaftStoreError::RegionNotFound(_)) => Ok(()),
+            Err(RaftStoreError::Transport(DiscardReason::Full)) => {
+                CHANNEL_FULL_COUNTER_VEC
+                    .with_label_values(&[&store_id.to_string()])
+                    .inc();
+                Ok(())
```

Comment:
Should return `Err(Error::from(e))`.

Reply:
do you mean something like `Err(Error::from(RaftStoreError::Transport(DiscardReason::Full)))` ?

Also confused about why the `RegionNotFound` error is being silenced & Discard is not.

Reply:
`RegionNotFound` can happen in normal condition, ignore it to avoid building new connections.